extensions/wikia/Recirculation/templates/client/Recirculation_sponsoredContent.mustache

```diff
@@ -12,7 +12,7 @@
 			{{title}}
 		</span>
 		<span class="mcf-card-article__subtitle">
-			Sponsored by {{siteName}}
+			{{attributionLabel}} {{attribution}}
```

Comment:
Can we just in case make the attributionLabel visible only if attribution is not empty?